modules/architecture-overview.adoc

```diff
@@ -7,16 +7,16 @@
 
 With {product-title} v4, the core story remains unchanged: {product-title} offers
 your developers a set of tools to evolve their applications under operational oversight
-and using Kubernetes to provide application infrastructure. The key change to v4 is
+and using Kubernetes to provide application infrastructure. The key change to {product-title}4 is
```

Comment:
@kalexand-rh minor typo - space missing.

Reply:
Thank you! https://github.com/openshift/openshift-docs/pull/13604